presto-hive-hadoop2/bin/run_on_docker.sh

```diff
@@ -81,7 +81,9 @@ docker-compose -f "${DOCKER_COMPOSE_LOCATION}" down || true
 trap termination_handler INT TERM
 
 # pull docker images
-docker-compose -f "${DOCKER_COMPOSE_LOCATION}" pull
```

Comment:
Why is this only needed on CI? Will this break when run on a laptop (or require a manual step)?

Reply:
I would revert this. This may cause stupid issues because of different images in CI and locally which could hard to diagnose.

When running tests locally and when images are downloaded it should not take long.

Reply:
We have the same switch in `run_on_docker.sh` for product test. This switch is needed to allow local testing of the images. As the image is built locally - it doesn't push it immediately to the docker hub.